crates/simulations/src/cozy/configs/base/config_params.yaml

```diff
@@ -0,0 +1,10 @@
+set_config_params:
+  leverage_factor: 10000
+  deposit_fee: 0
+market_config_params:
+  - weight: 5000
+    purchase_fee: 0
+    sale_fee: 0
+  - weight: 5000
+    purchase_fee: 0
+    sale_fee: 0
```

Comment:
are you manually updating these and running the sim to test with different configs?